File: src/SolucionesJava/Solucion_8.java
package SolucionesJava;

import java.util.Scanner;

public class Solucion_8 {
    public static void solucion_8() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("dame una palabra o frase para contarle las vocales");
        String frase = scanner.nextLine();
        int contador = 0;
        for (char i : frase.toCharArray()) {
            if (i == 'a' || i == 'e' || i == 'i' || i == 'o' || i == 'u') {
                contador++;
            }
        }
        System.out.println("el numero total de vocales que hay son " + contador);
    }
}

File: src/SolucionesKotlin/Solucion_8.kt
package SolucionesKotlin

fun Solucion_8() {
    println("dame una palabra o frase para contarle las vocales")
    val frase = readln()
    var contador = 0
    for (i in frase){
        if (i == 'a' || i == 'e'|| i == 'i'|| i == 'o'|| i == 'u'){
            contador +=1
        }
    }
    println("el numero total de vocales que hay son $contador")
}